prusa/link/web/lib/files.py

```diff
@@ -89,10 +115,15 @@ def fill_printfile_data(path: str, os_path: str, storage: str):
         meta.load_from_path(path)
         result['refs'] = sdcard_refs()
 
-    result['meta'] = meta.data
-    result['meta']['estimated_print_time'] = estimated_to_seconds(
+    estimated = estimated_to_seconds(
         meta.data.get('estimated printing time (normal mode)', ''))
-    result['display_name'] = basename(path)
+
```

Comment:
i have to discuss it with Michal and Ilja first

Reply:
Yes, this is just a preparation, I can change it